src/r_main.c

```diff
@@ -558,7 +558,7 @@ void R_ExecuteSetViewSize (void)
         }
     }
 
-    HU_disable_all_widgets();
+    //HU_disable_all_widgets();
```

Comment:
@fabiangreffrath Why do we need this line?

Reply:
I don't remember anymore. Had something to do with dynamic hud element arrangement.